README.md

```diff
@@ -62,6 +62,18 @@ Here is an example for the dialplan:
     exten => _X.,1,Dial(Dongle/r1/${EXTEN})
     exten => _X.,n,Hangup
 
+    exten => *#123#,1,DongleSendUSSD(dongle0,${EXTEN})
```

Comment:
I got this message for sending USSD from xlite sip client. From astrisk cli ussd send without problem.

[2018-08-17 23:58:26] WARNING[12618][C-00000001]: pbx.c:2866 pbx_extension_helper: No application 'DongleSendUSSD' for extension (dongle-trunk, *2#, 1)
  == Spawn extension (dongle-trunk, *2#, 1) exited non-zero on 'SIP/1007-00000001'